flink-libraries/flink-table/src/main/scala/org/apache/flink/table/codegen/calls/BuiltInMethods.scala

```diff
@@ -135,4 +135,6 @@ object BuiltInMethods {
   val FROMBASE64 = Types.lookupMethod(classOf[ScalarFunctions], "fromBase64", classOf[String])
 
   val TOBASE64 = Types.lookupMethod(classOf[ScalarFunctions], "toBase64", classOf[String])
+
+  val LTRIM = Types.lookupMethod(classOf[ScalarFunctions], "lTrim", classOf[String])
```

Comment:
Use `ltrim` function in calcite directly. 
`val LTRIM = Types.lookupMethod(classOf[SqlFunctions], "ltrim", classOf[String])`